administrator/components/com_users/src/Controller/UserController.php

```diff
@@ -117,6 +117,12 @@ public function save($key = null, $urlVar = null)
             $this->setRedirect($return);
         }
 
+        // If a user has to renew a password but has no permission for users
+        if (!$this->app->getIdentity()->authorise('core.admin', 'com_users'))
+        {
```

Comment:
```suggestion
        if (!$this->app->getIdentity()->authorise('core.admin', 'com_users')) {
```
Fix PHPCS failing in drone.